cfgov/jinja2/v1/_layouts/base.html

```diff
@@ -190,8 +190,15 @@
     {# Check and include template-level JavaScript. #}
     {% include '/js/routes/' + request.path.split('/')[1] + '/single.js' ignore missing %}
 
+    {# TODO: Remove conditional when this is fully moved to wagtail. #}
     {# Check and include page-level JavaScript. #}
-    {% include '/js/routes/' + request.path[1:] + 'index.js' ignore missing %}
+    {% if page %}
+        {% for js in page.media %}
```

Comment:
@sebworks hmm sounds good but i'd rather not go down the integration path just yet. i'll need time to access the pro/cons between the many different loaders out there in a future sprint